### ISSUE TITLE
Body writes System.Collections.hashtable

### ISSUE THREAD

**Issue by lboening** [EsOsO/Logging#38]
Problem: The argument for the -body is processed as System.Collections.Hashtable

Inputs: 
```powershell
Set-LoggingDefaultFormat -Format '[%{timestamp:+yyyy-MM-dd HH:mm:ss.zzz}][%{level:-7}][%{caller}][%{message}][%{lineno}] %{body}'

Write-Log -Level ERROR -Message 'Hello, {0}!' -Arguments 'World' -Body @{Server='srv01.contoso.com'}
```

Expected: 
[2019-04-01 12:12:07.-05:00][ERROR  ][Deploy-PowerShellAppV2.ps1][Hello, World!][1527] [server='srv01.contoso.com']

Result (error):
[2019-04-01 12:12:07.-05:00][ERROR  ][Deploy-PowerShellAppV2.ps1][Hello, World!][1527] System.Collections.Hashtable

Help.

**Comment by EsOsO** [comment]
`-Body` is meant to be used with the file target, I suggest you to set logging format per target to avoid problems:

```powershell
Add-LoggingTarget -Name Console -Configuration @{
  Level='DEBUG'
  Format='[%{timestamp:+yyyy-MM-dd HH:mm:ss.zzz}][%{level:-7}][%{caller}][%{message}][%{lineno}]'
}

Add-LoggingTarget -Name File -Configuration @{
  Level='DEBUG'
  Format='[%{timestamp:+yyyy-MM-dd HH:mm:ss.zzz}][%{level:-7}][%{caller}][%{message}][%{lineno}] %{body}'
  PrintBody=$true
  Path='C:\temp\tmp.log'
}
```

**Comment by lboening** [comment]
I tried that and passed a hashtable. The result is still Systems.Collection.Hashtable.

In the file.ps1 target, I see these lines:

```powershell
        if ($Configuration.PrintBody -and $Log.Body) {
            $Text += ': {0}' -f ($Log.Body | ConvertTo-Json -Compress)
        }
```
Could it be that a $log.body that is a hashtable needs a "getenumerator()" method?

**Comment by EsOsO** [comment]
@lboening you're right, the issue is on line L21 of File.ps1, where i format the message.
I'll take a look asap

**Comment by EsOsO** [comment]
Should be fixed in 2.10.0.

Let me know if persists
